### DISCUSSION TITLE
Find all references by LSP (like that in vscode with rust-analyzer)

### DISCUSSION THREAD

**Discussion by zhaojiangbin** [helix-editor/helix#1681] [Q&A] [answered]
Does helix have it? It appears that the global search in workspace does not query the LSP.

I have found this feature useful.

**Comment by kirawi** [comment, marked as answer]
https://docs.helix-editor.com/keymap.html#goto-mode 
`g` `r`

**Reply by zhaojiangbin** [reply]
Ah, nice. Thanks!
